lua/entities/gmod_wire_gpu/cl_gpuvm.lua

```diff
@@ -107,6 +107,17 @@ function ENT:OverrideVM()
     end
   end
 
+  self.VM.ExternalWrite = function(VM, Address, Value)
+    if (Address >= 65536) and (Address <= 131071) then
+      return true
+    elseif Address < 0 then
+      VM:WritePort(Address, Value)
```

Comment:
Ah wait, shouldn't this be Address+65536?

Reply:
I don't think so, but then again I put that in without checking how that function's called.

Reply:
Yeah, I messed that one up. Address+65536 isn't correct though - that would end up overwriting other internal registers.